src/main/resources/application-console.yml

```diff
@@ -9,6 +9,17 @@ opa:
     url: http://localhost:8081
   query: "data.contentcloud.api.allow == true"
 
+content-cloud:
```

Comment:
This does not match with the `@ConfigurationProperties` annotation on  `CorsResolverProperties`, so I think this will not work.